src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -11,7 +11,8 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    //public final String feedbackToUser;
+	private final String feedbackToUser // Changed variable from public to private
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.